File: Daily Test/Java/Match Name And Designation - Employees 31.1.2026.java
import java.util.*;
import java.io.*;

public class Hello {
    public static void main(String[] args) throws Exception {
        Scanner sc = new Scanner(System.in);

        String file1 = sc.nextLine();
        String file2 = sc.nextLine();

        Map<Integer, String> nameMap = new HashMap<>();
        Map<Integer, String> desMap = new HashMap<>();

        BufferedReader br1 = new BufferedReader(new FileReader(file1));
        int n = Integer.parseInt(br1.readLine().trim());
        for (int i = 0; i < n; i++) {
            String[] parts = br1.readLine().split(" ");
            nameMap.put(Integer.parseInt(parts[0]), parts[1]);
        }
        br1.close();

        BufferedReader br2 = new BufferedReader(new FileReader(file2));
        br2.readLine();
        for (int i = 0; i < n; i++) {
            String[] parts = br2.readLine().split(" ");
            desMap.put(Integer.parseInt(parts[0]), parts[1]);
        }
        br2.close();

        List<Integer> ids = new ArrayList<>(nameMap.keySet());
        Collections.sort(ids);

        for (int id : ids) {
            System.out.println(id + " " + nameMap.get(id) + " " + desMap.get(id));
        }
    }
}

File: Daily Test/Python/Match Name And Designation - Employees 31.1.2026.py
file1 = input().strip()
file2 = input().strip()

name_map = {}
des_map = {}

with open(file1) as f:
    n = int(f.readline())
    for _ in range(n):
        id, name = f.readline().split()
        name_map[int(id)] = name

with open(file2) as f:
    f.readline()
    for _ in range(n):
        id, des = f.readline().split()
        des_map[int(id)] = des

for id in sorted(name_map):
    print(id, name_map[id], des_map[id])